Sources/Testing/Running/Runner.swift

```diff
@@ -103,21 +103,18 @@ extension Runner {
   ///
   /// - Parameters:
   ///   - sequence: The sequence to enumerate.
-  ///   - step: The plan step that controls parallelization. If `nil`, or if its
-  ///   ``Runner/Plan/Step/action`` property is not of case
-  ///   ``Runner/Plan/Action/run(options:)``, the
-  ///   ``Configuration/isParallelizationEnabled`` property of this runner's
-  ///   ``configuration`` property is used instead to determine if
-  ///   parallelization is enabled.
+  ///   - configuration: The configuration with which the elements of `sequence`
+  ///     should be enumerated. This function uses the configuration to
+  ///     determine whether parallelization is enabled. The default value is the
+  ///     current configuration, if any, or else the default configuration.
   ///   - body: The function to invoke.
   ///
   /// - Throws: Whatever is thrown by `body`.
-  private func _forEach<E>(
+  private static func _forEach<E>(
     in sequence: some Sequence<E>,
-    for step: Plan.Step?,
+    configuration: Configuration = .current ?? .init(),
```

Comment:
Do we ever actually explicitly pass `configuration`? If not, maybe just make it a local in the function(s) that use it?

Reply:
Can do